### DISCUSSION TITLE
How can i use the old vm creation screen?

### DISCUSSION THREAD

**Discussion by zx331645** [utmapp/UTM#4550] [UTM Help (macOS)]
This one
![image](https://user-images.githubusercontent.com/63444823/197366951-7776c225-caf7-4aac-a4b6-4e5f4291bf03.png)

**Comment by JezerM** [comment]
Open your VM settings and go to the QEMU tab. Scroll down to the bottom and there you will be able to modify your QEMU arguments.

<img width="1148" alt="Captura de Pantalla 2022-10-22 a la(s) 19 29 13" src="https://user-images.githubusercontent.com/59768785/197368896-9fbb501b-3774-46db-a60b-b13fe581ac83.png">
